README.md

```diff
@@ -0,0 +1,51 @@
+# Aragon Court Dashboard <a href="https://aragon.org/"><img align="right" src="docs/assets/LogoAccent.svg" height="80px" /></a>
+
+### Aragon Court handles subjective disputes that require the judgment of human jurors.
+
+<a href="https://court.aragon.org/dashboard"><img src="docs/assets/screenshot.png" /></a>
```

Comment:
Only thing I would say for this screenshot is to make the screen a bit thinner, so less of the background whitespace shows :).